src/ViewHelpers/IfViewHelper.php

```diff
@@ -12,36 +12,50 @@
 
 /**
  * This ViewHelper implements an if/else condition.
+ * The code entered within this ViewHelper is then only executed
+ * or shown, when the condition matches.
```

Comment:
This sounds a bit too fuzzy to me. Not sure if this addition is helpful.

Apart from that, I would replace `when` with `if` because it has nothing to do with time. :)